docs/examples/deep_corrector.md

```diff
@@ -0,0 +1,123 @@
+Cornell Movie Dialog dataset at `/input` path. You will notice that other parameters use files
+mounted in this path.
+- The data name [floydhub/datasets/deep-text-corrector/1](https://www.floydhub.com/floydhub/datasets/deep-text-corrector/1)
+points to the pre-processed dataset on FloydHub.
+- The job is running on a gpu instance (Because of the `--gpu` flag).
+- This project uses Tensorflow-0.12 installed on Python 2. (See the `--env` flag)
+
+This job takes about 10 minutes to run and generate a model. You can follow along the progress
+by using the [logs](../commands/logs.md) command.
+
+```bash
+$ floyd logs <JOB_NAME> -t
+```
+
+Floyd saves any content stored in the `/output` directory after the job is
+finished. This output can be used as a datasource in the next project.  To get
+the name of the output generated by your job use the
+[info](../commands/info.md) command.
+
+```bash
+$ floyd info <JOB_NAME>
+```
+
+
+## Evaluating
+
+To evaluate your model you can run the `correct_text.py` script with the `decode` flag.
+You need a file containing short messages for evaluation. The `test.txt` file already has some
+inputs. You can update or add more strings to this file - one per line. You also need to
+use the output from the training step above as the datasource in this step.
+
+```bash
+floyd run --env tensorflow-0.12:py2 --data <REPLACE_WITH_JOB_OUTPUT_NAME> "python correct_text.py --train_path /input/data/movie_dialog_train.txt --test_path test.txt --config DefaultMovieDialogConfig --data_reader_type MovieDialogReader --input_path /input --decode"
+```
+
+You can track the status of the run with the status or logs command. The logs should print the
+concerted messages from the test.txt file.
+
+```bash
+$ floyd status <JOB_NAME>
+$ floyd logs <JOB_NAME> -t
+```
+
+
+## Improving your model
+
+You may notice that the output does not look great. In fact, the algorithm would've added more
+mistakes into the sentences than correct it. That is because we ran the training for a small number
+of iterations. To train a fully working model try the training step again, this time by setting
+the flag `num_steps` to a large value. In general, about 20000 steps are necessary to give a
+working corrector model. (Note: This takes a few hours to run on the GPU instance)
+
+## Evaluate pre-trained models
+
+If you want to try out a pre-trained model, FloydHub has a public job output for
+this. You can mount it with job output name:
+[floydhub/deep-text-corrector/23/output](https://www.floydhub.com/floydhub/projects/deep-text-corrector/23/output)
+.
+
+```bash
+floyd run --env tensorflow-0.12:py2 --data floydhub/deep-text-corrector/23/output "python correct_text.py --train_path /input/data/movie_dialog_train.txt --test_path test.txt --config DefaultMovieDialogConfig --data_reader_type MovieDialogReader --input_path /input --decode"
+```
+
+This model should perform better on the given inputs compared to the previous one.
+
+
+## Rest API
+
+Coming Soon!
```

Comment:
I saw that you have the app.py file for serving, can you add the command for running that here?